constructor/_schema.py

```diff
@@ -828,6 +830,13 @@ class ConstructorConfiguration(BaseModel):
     Use the standalone binary to perform the uninstallation on Windows.
     Requires conda-standalone 24.11.0 or newer.
     """
+    frozen_file: dict | None = None
+    """
+    Protect the base environment with a `frozen` marker file. Requires conda 25.7.0 or newer. This setting can be:
```

Comment:
```suggestion
    Protect the base environment with a `frozen` marker file. Requires conda 25.5.0 or newer. This setting can be:
```

`conda` added frozen environments support at version 25.5.0.

Reply:
Also, should we add the CEP here?

Reply:
True, but I thought `conda-standalone 25.5.x` fails with protected environments and it was related to conda not having the full implementation?

Reply:
Yes, `conda-standalone 25.5.x` fails during the installation process because `conda 25.5.0` didn't properly process the `CONDA_PROTECT_FROZEN_ENVS` variable, which is important for the installation. However, for the *feature* to work, you need `conda 25.5.0`. So, maybe write "requires `conda 25.5.0` or newer in the `base` environment"?